lib/widgets/resources/board.dart

```diff
@@ -34,6 +36,9 @@ class _ViamBoardWidgetState extends State<ViamBoardWidget> {
   final _pwmFreqFormKey = GlobalKey<FormState>();
   int pwmFrequency = 0;
 
+  Duration refreshInterval = const Duration(seconds: 1);
```

Comment:
What do you think about making this a property of the widget? That way a consumer could specify the refresh interval themself?

Reply:
DO you think it would be valuable to default it to 1 second though?